README.md

```diff
@@ -1,2 +1,39 @@
 # hello-world
 Obligatory tutorial
+
+Here is some example text. Isn't it exciting? 
+Make and commit changes
+
+    Click the README.md file.
+    Click the 
+
+_pencil_ *icon* **in the** upper right corner of the file view to edit.
+In the editor, write a bit about yourself.
+<code>Write a commit message that describes your changes.
+Click Commit changes button.</code>
+
+Here are just some examples.
+
+Random Blynk code:
+
+```
+/* Comment this out to disable prints and save space */
+#define BLYNK_PRINT Serial
+
+
+#include <ESP8266WiFi.h>
+#include <BlynkSimpleEsp8266_SSL.h>
+
+// You should get Auth Token in the Blynk App.
+// Go to the Project Settings (nut icon).
+char auth[] = "YourAuthToken";
+
+// Your WiFi credentials.
```

Comment:
Don't forget that, for local blynk server, I need to add ssid/pass/auth/host not just the first three.